ref/Microsoft.Build.Tasks.Core/net/Microsoft.Build.Tasks.Core.cs

```diff
@@ -807,8 +807,10 @@ public partial class RemoveDuplicates : Microsoft.Build.Tasks.TaskExtension
     {
         public RemoveDuplicates() { }
         [Microsoft.Build.Framework.OutputAttribute]
-        public Microsoft.Build.Framework.ITaskItem[] Filtered { get { throw null; } set { } }
-        public Microsoft.Build.Framework.ITaskItem[] Inputs { get { throw null; } set { } }
+        public Microsoft.Build.Framework.ITaskItem[] Filtered { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute]set { } }
```

Comment:
These changes were generated as part of the build. I assume they are to be committed.

Reply:
Yes, please. These (generated) files are our public API surface. We like to keep them checked in so that we can explicitly consider any diff to the public API.

Here, I think it felt the need to add `CompilerGeneratedAttribute` because of the switch to autoproperties, which isn't particularly useful, but it's nice to see the new field with the OutputAttribute.